rest_framework-stubs/generics.pyi

```diff
@@ -19,12 +19,19 @@ def get_object_or_404(
 class UsesQuerySet(Protocol[_MT_co]):
     def get_queryset(self) -> QuerySet[_MT_co]: ...
 
+# Can't just use BaseFilterBackend because there's also things like django_filters.rest_framework.DjangoFilterBackend that are
+# valid options but don't extend it
+class BaseFilterProtocol(Protocol[_MT_co]):
```

Comment:
Maybe you should create a new type var to please the type tests? Check out logs to see the suggestions.

Reply:
Works now :)